src/com/company/Sendable.java

```diff
@@ -0,0 +1,9 @@
+package com.company;
+
+public static interface Sendable<T> {
```

Comment:
do we need this interface here?

Reply:
Yes, i use interface for structure. It is convenient for me to use

Reply:
I can delete it if it's mistake

Reply:
for this task this interface is unnecessary